Docker.DotNet/Models/ExtractArchiveToContainerParameters.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Docker.DotNet.Models
+{
+    public class ExtractArchiveToContainerParameters
+    {
+        [QueryStringParameter("path", true)]
+        public string Path { get; set; }
+
+        [QueryStringParameter("noOverwriteDirNonDir", false, typeof(BoolQueryStringConverter))]
+        public bool NoOverwriteDirNonDir { get; set; }
```

Comment:
It appears like this argument is not required. Can we make this field `bool?` type so that it's omitted unless explicitly set by the user?